qcodes/tests/test_station.py

```diff
@@ -549,15 +549,15 @@ def test_setup_delegate_parameters():
         label: ch1
         scale: 1
         offset: 0
-        limits: -10, 10
+        limits: [-10, 10]
     add_parameters:
       gate_a:
         source: ch1
         unit: mV
         label: main gate
         scale: 2
         offset: 1
-        limits: -6, 6
+        limits: [-6, 6]
```

Comment:
shall we have these as floats for the sake of "covering more cases"?
```suggestion
        limits: [-6.0, 6.0]
```

Reply:
Good point. Let me also change the check then.